vividus-engine/src/main/java/org/vividus/steps/VariableResolver.java

```diff
@@ -142,11 +142,20 @@ private Object resolveVariable(String value, String variableKey)
 
     private Object replaceVariableKeyWithValue(String target, String variableKey)
     {
-        Object variableValue = getVariableValue(variableKey);
-        if (variableValue == null)
+        Object variableValue;
+        try
         {
+            variableValue = getVariableValue(variableKey);
+        }
+        catch (IllegalArgumentException e)
```

Comment:
we should never catch runtime exceptions (especially very generic `IllegalArgumentException`)